src/core/connection.ts

```diff
@@ -138,16 +138,20 @@ export class Connection extends EventEmitter {
       return this.getKeys(match, count, keysAry, parseInt(newCursor));
     }
 
+
     this.emit(
       "error",
       new Error(
         "You must establish a connection to redis before running the getKeys command."
       )
     );
+
+    return null;
```

Comment:
this is not typing change, this changes runtime code.

Reply:
It's just on the end of the function, so just remove the implicite return, 
"noImplicitReturns"

This rule activate, show that in this function some return was missing

Reply:
I suggest submitting code fixes as a separate PR.